plugins/w3c_logs.yaml

```
@@ -0,0 +1,109 @@
+version: 0.0.1
+title: W3C
+description: File Input W3C Parser
+parameters:
+  - name: file_log_path
+    type: "[]string"
+    required: true
+    default: 
+      - "/var/log/zookeeper/zookeeper.log"
+  - name: exclude_file_log_path
+    type: "[]string"
+    default: 
+      - ""
```

This default should be an empty list
```suggestion
    default: []
```